Agents/ForecastingAgent/tests/conftest.py

```diff
@@ -143,11 +147,29 @@
 TEST_CASE_10 = 'TFT_OM_Quantity_with_Measure_with_Unit__no_overwriting'
 TEST_CASE_11 = 'TFT_OM_Quantity_without_Measure_with_Unit__overwriting'
 TEST_CASE_12 = 'TFT_OM_Quantity_without_Measure_with_Unit__no_overwriting'
+# Prophet with covariates test cases
+TEST_CASE_13 = 'Prophet_Covariates_OM_Quantity_without_scaling_with_Measure_without_Unit__overwriting'
+TEST_CASE_14 = 'Prophet_Covariates_OM_Quantity_without_scaling_with_Measure_without_Unit__no_overwriting'
+TEST_CASE_15 = 'Prophet_Covariates_OM_Quantity_without_scaling_without_Measure_with_Unit__overwriting'
+TEST_CASE_16 = 'Prophet_Covariates_OM_Quantity_without_scaling_without_Measure_with_Unit__no_overwriting'
+TEST_CASE_17 = 'Prophet_Covariates_OM_Quantity_with_scaling_with_Measure_without_Unit__overwriting'
+TEST_CASE_18 = 'Prophet_Covariates_OM_Quantity_with_scaling_with_Measure_without_Unit__no_overwriting'
+TEST_CASE_19 = 'Prophet_Covariates_OM_Quantity_with_scaling_without_Measure_with_Unit__overwriting'
+TEST_CASE_20 = 'Prophet_Covariates_OM_Quantity_with_scaling_without_Measure_with_Unit__no_overwriting'
 DERIVATION_INPUTS_5 = [IRI_TO_FORECAST_1, FORECASTING_MODEL_2,
                        FC_INTERVAL_1, FC_FREQUENCY_1, HIST_DURATION_2]
 DERIVATION_INPUTS_6 = [IRI_TO_FORECAST_2, FORECASTING_MODEL_2,
                        FC_INTERVAL_1, FC_FREQUENCY_1, HIST_DURATION_2]
+DERIVATION_INPUTS_7 = [IRI_TO_FORECAST_1, FORECASTING_MODEL_3,
+                       FC_INTERVAL_1, FC_FREQUENCY_1, HIST_DURATION_2]
+DERIVATION_INPUTS_8 = [IRI_TO_FORECAST_2, FORECASTING_MODEL_3,
+                       FC_INTERVAL_1, FC_FREQUENCY_1, HIST_DURATION_2]
+DERIVATION_INPUTS_9 = [IRI_TO_FORECAST_1, FORECASTING_MODEL_4,
+                       FC_INTERVAL_1, FC_FREQUENCY_1, HIST_DURATION_2]
+DERIVATION_INPUTS_10 = [IRI_TO_FORECAST_2, FORECASTING_MODEL_4,
+                       FC_INTERVAL_1, FC_FREQUENCY_1, HIST_DURATION_2]
 COVARIATES_1 = [AIRTEMPERATURE_1, ISHOLIDAY_1]
+COVARIATES_2 = [PROMOTION_1, SPECIALEVENT_1]
```

Comment:
Currently we test 0 and 2 covariates. I suggest we also include the most likely case of 1 covariate